bool is_attach  # true to attach, false to detach
string odom_topic # topic of the odometry message to request to subscribe to and attach to